app/app.go

```diff
@@ -666,6 +643,39 @@ func NewStrideApp(
 		app.AccountKeeper, app.BankKeeper, app.DistrKeeper, app.StakingKeeper,
 	)
 
+	// Add wasm keeper
+	wasmCapabilities := "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4"
```

Comment:
In the go mod it looks like it is using wasm 1.5.2 so does this mean we should add wasmCapabilities cosmwasm_1_5 as well?

Reply:
I don't think there is a 1_5

https://github.com/CosmWasm/wasmd?tab=readme-ov-file#for-contract-developers